src/Controller.cpp

```diff
@@ -112,3 +118,38 @@ void Controller::add_statistics_data(
         end_time_default,
         backend::string_to_statistic_kind(statistic_kind));
 }
+
+QString Controller::fastdds_version()
+{
+    return utils::to_QString(FASTRTPS_VERSION_STR);
+}
+
+QString Controller::fastdds_statistics_backend_version()
+{
+    return utils::to_QString(FASTDDS_STATISTICS_BACKEND_VERSION_STR);
+}
+
+QString Controller::qt_version()
+{
+    return QT_VERSION_STR;
```

Comment:
It does not require an include? `qconfig.h`

Reply:
` #include <QtGlobal> `